Bottom: 4a94f0a0a0345f134da23d76e34447998bd3437c
Top:    d3ec6b8ac4e21608b10cc1493a58f76330c2aad7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-26 23:30:43 +0000

hid/gtk: Remove unused file scope variables event_x and event_y


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 569a09b..e0120b6 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -214,8 +214,6 @@ ghid_port_ranges_zoom (gdouble zoom)
  * handles all events from PCB drawing area
  */
 
-static gint event_x, event_y;
-
 void
 ghid_get_coords (const char *msg, int *x, int *y)
 {
@@ -231,14 +229,13 @@ ghid_get_coords (const char *msg, int *x, int *y)
 gboolean
 ghid_note_event_location (GdkEventButton * ev)
 {
-  gint x, y;
+  gint event_x, event_y;
   gboolean moved;
 
   if (!ev)
     {
-      gdk_window_get_pointer (ghid_port.drawing_area->window, &x, &y, NULL);
-      event_x = x;
-      event_y = y;
+      gdk_window_get_pointer (ghid_port.drawing_area->window,
+                              &event_x, &event_y, NULL);
     }
   else
     {
